libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2000,14 +2000,16 @@ void WaveTrack::PasteOne(
         for (const auto& clip : track.mClips)
             if (clip->GetPlayStartTime() > t0 - (1.0 / rate))
                 clip->ShiftBy(insertDuration);
-    } else if (split)
+    }
+    else
     {
-       track.SplitAt(t0);
+       if (!merge)
```

Comment:
This repairs the build failure of two commits previous

Reply:
So, behavior may change in the case editing can't move clips, and merge is true.  The lines after this small-scope `if` happen when they did not happen before.